### PR TITLE
Update files

### PR DESCRIPTION
**Pull request by ocefpaf**
I thinks this should fix the source tarball. See https://github.com/conda-forge/cf_units-feedstock/pull/14 for the failures we are getting with the latest PyPI tarball.

### CODE REVIEW

File: MANIFEST.in [hunk 1]
```diff
@@ -1,7 +1,6 @@
-include *.rst
```
Comment (ocefpaf) [inline MANIFEST.in:1]: I'm counting on #103 for this change :smile:

File: cf_units/etc/site.cfg.template [hunk 2]
```diff
@@ -1,2 +0,0 @@
-[System]
```
Comment (pelson) [inline cf_units/etc/site.cfg.template]: Technically, this is still valid (XML file, not ``.so``).

File: MANIFEST.in [hunk 3]
```diff
@@ -1,7 +1,6 @@
...
 include *.txt
 recursive-include cf_units *.py
-include cf_units/etc/site.cfg.template
+include cf_units/etc/_udunits2.pxd
```
Comment (pelson) [inline MANIFEST.in]: etc?

Could you simply remove these lines and update the recursive include:

```
recursive-include cf_units *.py *.pxd *.pyx
```

**Review by pelson — APPROVED** [review]

File: MANIFEST.in (hunk 1 above)
Comment (pelson) [reply MANIFEST.in:1]: Though, it will need to be replaced with ```*.md```...

**Comment by pelson** [comment]
I've fixed this locally with:

```
--- a/MANIFEST.in
+++ b/MANIFEST.in
@@ -1,6 +1,6 @@
 include *.rst
 include *.txt
-recursive-include cf_units *.py
+recursive-include cf_units *.py *.pxd *.pyx
 include cf_units/etc/site.cfg.template
 include CHANGES COPYING COPYING.LESSER INSTALL
 include versioneer.py
```

(tested and 👍)

File: cf_units/etc/site.cfg.template (hunk 2 above)
Comment (ocefpaf) [reply cf_units/etc/site.cfg.template]: Oops, yep! (I guess I need to fix our conda recipe too.)

**Comment by pelson** [comment]
Excellent! Thanks @ocefpaf.

File: MANIFEST.in (hunk 1 above)
Comment (pelson) [reply MANIFEST.in:1]: P.S. You have write access on this repo @ocefpaf. Why count on it, and not feel free to just merge 😜.

Comment (ocefpaf) [reply MANIFEST.in:1]: Will do for simpler things, but it turns out I needed a review on this one after all.

**Pull request merged**